weblate/api/tests.py

```diff
@@ -292,9 +292,18 @@ def test_create_component(self):
                 'repo': self.format_local_path(self.git_repo_path),
                 'filemask': 'po/*.po',
                 'file_format': 'po',
+                'push': 'git@github.com:example/push.git',
             },
         )
         self.assertEqual(Component.objects.count(), 2)
+        self.assertEqual(
+            Component.objects.all()[1].push,
```

Comment:
I don't think this is safe, no ordering is guaranteed here.

Reply:
Done. I replaced this with more predictable behavior
```python
Component.objects.get(
                slug='api-project',
                project__slug='test'
            ).push
```